indico/modules/events/management/templates/_settings.html

```diff
@@ -118,12 +123,12 @@
             <dd class="break-word">{{ with_default(event.map_url) }}</dd>
         </dl>
 
-        {% if wallet_location_warning %}
+        {% if wallet_location_warning or pass_location_warning %}
             <br>
             {% call message_box('warning') %}
-                {% trans %}
-                    Google Wallet ticketing is enabled in this event and you specified a venue/room or address, but
-                    not both. For the ticket in Google Wallet to display information about the event's physical
+                {% trans ticket_names=ticket_names|join(' and ')%}
+                    {{ ticket_names }} ticketing is enabled in this event and you specified a venue/room or address, but
```

Comment:
yes, i prefer less "magic" there. BTW, does apple pass have this same restriction to begin with? or is this only a problem with google wallet?

Reply:
Neither Google Wallet nor Apple Pass have any restrictions about the venue name and address: only venue name is perfectly acceptable, and that was the way I applied in my early code.
The need to have both has been your call @ThiefMaster .

Reply:
That is not correct. I tested and both Google's web interface and their API reject changes containing only one of those fields. They are simply ignored.

Reply:
Sorry my bad, you are right @ThiefMaster .
I've checked the Apple Pass: there is no issue for the venue/address because there is no specific content check: 
in the ticket we define the Label and the Value, so we can put whatever we want:

`ticket.addSecondaryField('event-venue', venue_name, 'Venue')
ticket.addBackField('back-event-venue', venue_name, 'Venue')`

Reply:
I think it's still a good idea that we require the same constraints for both Google and Apple integrations. What do you think @ThiefMaster?

Reply:
Considering that this is still a mess regarding i18n I would change this and only show  the warning in case of Google.

Reply:
Done.